packages/contractkit/src/wrappers/Governance.ts

```diff
@@ -183,7 +184,7 @@ export class GovernanceWrapper extends BaseWrapper<Governance> {
     })
   )
 
-  static toParams = (proposal: Proposal): ProposalParams => {
+  static proposalToParams = (proposal: Proposal): ProposalParams => {
```

Comment:
nit: this static function can also be utility function on the module but outside the class